TeachingRecordSystem/src/TeachingRecordSystem.Api/V3/VNext/Requests/SetPiiRequest.cs

```diff
@@ -0,0 +1,14 @@
+using TeachingRecordSystem.Core.ApiSchema.V3.V20250203.Dtos;
+
+namespace TeachingRecordSystem.Api.V3.VNext.Requests;
+
+public record SetPiiRequest
+{
+    public required string FirstName { get; init; }
```

Comment:
Do the required properties here align with the required properties for TRN allocation?